Makefile

```diff
@@ -8,9 +8,9 @@ BUILDTAGS ?= ""
 BUILDER ?= "docker"
 EXTRA_BUILD_ARGS ?= ""
 
-CERT_MANAGER_VERSION ?= v1.12.0
-CONTROLLER_GEN_VERSION ?= v0.12.0
-GOLANGCI_LINT_VERSION ?= v1.52.2
+CERT_MANAGER_VERSION ?= v1.12.2
+CONTROLLER_GEN_VERSION ?= v0.12.1
```

Comment:
`make generate` is needed if you do this.

Reply:
Ah, yeah, I forgot. Thanks!